rpcs3/main.cpp

```diff
@@ -154,26 +154,39 @@ struct pause_on_fatal final : logs::listener
 {
 	~pause_on_fatal() override = default;
 
-	void log(u64 /*stamp*/, const logs::message& msg, const std::string& /*prefix*/, const std::string& /*text*/) override
+	void log(u64 /*stamp*/, const logs::message& msg, const std::string& prefix, const std::string& text) override
```

Comment:
Do not change this function 👎 
It's the pause_on_fatal logs::listener. It's only supposed to pause on fatal...
Ideally you would write a new logs listener that's called console_writer or something similar.
This new writer would only be created conditionally, since it will have an impact on performance.
Maybe there's an even simpler solution.

If you're gonna change anything related to logging, then you should make a dedicated branch/pr.